client/src/styles/global.scss

```diff
@@ -185,12 +227,25 @@ $primary-color: #112f4e;
     margin: 0;
   }
 
-  padding-top: 7px;
-  padding-bottom: 7px;
-  min-height: 4em;
+  min-height: 2.5em;
+  margin: 0 !important;
 
-  * + .usa-alert {
+  .usa-alert {
     margin: 0 !important;
```

Comment:
Kind of worried that we need to override everything but that isn't really in your PR, more a comment that we will probably need to refactor custom styles at some point.

Reply:
100% agree! I think I managed to remove it in this case, but there are other places we should try to fix.